picard/ui/options/general.py

```diff
@@ -65,6 +66,7 @@ class GeneralOptionsPage(OptionsPage):
         ('analyze_new_files', ['analyze_new_files']),
         ('cluster_new_files', ['cluster_new_files']),
         ('ignore_file_mbids', ['ignore_file_mbids']),
+        (READTHEDOCS_UPDATES_ALLOWED_KEY, ['check_rtd_updates']),
```

Comment:
While it could be convenient sometimes, I think it is easier to read if we use the string, rather than a constant. Is this value supposed to change? I guess not much since that's an option key.

```python
        ('check_rtd_updates', ['check_rtd_updates']),
```

Also no need to import constants (for this at least)

What do you think?

Reply:
Okay, I'll make the change.  The only reason I set it up as a constant was because it was an easy way to make sure I was using the same key throughout during the development.  I ended up changing the key string a couple of times before landing on something that was sufficiently descriptive but wasn't overly long.  Now that it's set, I can enter it directly instead of using the constant.